### DISCUSSION TITLE
Adding third phase to reconstructed parent grains

### DISCUSSION THREAD

**Discussion by huseyinhzl** [ORTools4MTEX/ORTools#41] [Q&A] [answered]
Hi All,

I am using ORTools with MTEX 5.8.2 to do a parent grain reconstruction in titanium. Due to the change in its chemical composition, I have a third phase in the microstructure and I assume it is nothing to do with the parent grain orientations. When I run the parent grain reconstruction. I end up with white areas in the plot, which belongs to the third phase. Here, I am wondering how I can show my third phase in the reconstructed parent grains. Also, I have not indexed the area which I want to fill with. I have done the steps written in the MTEX document but it seems not working. So, I am wondering when I should do the filling, before creating new ebsdP command or after creating an ebsdP. P.S. I have nonconvex data so I am creating another ebsdP values

Thank you so much

I have put two images that I have created in MTEX. One is the phase plot (orange ones are my third phase) and the other is the calculated parent grains. Also, it seems some of the third phases has been eaten up during the parent grain reconstruction

![Phase plot](https://user-images.githubusercontent.com/120036622/235157656-3c394045-d623-4215-833d-238e8d97504c.png)

![Parent Grains](https://user-images.githubusercontent.com/120036622/235156838-a848ece4-9e4c-4663-ab0d-ab5092114d84.png)
802-005aec19-8d26-475c-a399-9ca8b04c81b3.png)

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @huseyinhzl,

In the figures you have shown, the first one is  phase map whereas the second appears to be an IPF map.
Combining these 2 ebsd map types into one map, as you wish to do, is not conventional practice.

For a demonstration on how to plot:
(i) parent, (ii) reconstructed parent, and (iii) any other phases 
in an **ORTools phase map** in 1 plot, 
please run [**ORTools Example 6**](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example6.m) up to line 76.

Once complete, please type in the following in the command line window:
```matlab
close all; plotMap_phases(job1)
``` 

The plotted **phase map** will comprise all phases in the ebsd map.
In this case, the 3 phases are:
(i) the parent gamma, 
(ii) the reconstructed parent epsilon from child alpha, and 
(iii) the remnant child alpha that was not reconstructed.

Hope this helps.

Warm regards,
Azdi
